telco-core/networking/sriov/SriovNetworkPoolConfig.yaml

```diff
@@ -0,0 +1,14 @@
+# optional
+# count: 0-N
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkPoolConfig
+metadata:
+  name: $name
+  namespace: openshift-sriov-network-operator
+spec: {} # $spec
+  # eg
+  #maxUnavailable: 2
```

Comment:
Is this something that we should set by default and allow the user to change as desired? (ie make this code active)

Reply:
I don't think so.

if there is no policy the operator will continue to work as before one node at a time and I would like to maintain that.

Reply:
not directly RDS related, but it would be nice if sriov was following (by default) the number from the MCP

Reply:
I didnt say always follow MCPs,
but allow to follow it (by default) instead of providing a value.
NM, can discuss offline